notebooks/06-cifar10-baseline.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>should it be <code>lightining_logs/version_0</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/pytorch-lightning/pull/4818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p><code>lightning_logs</code> is the default directory for tensorboard logs so we need to provide it as the path in order to see all versions and runs.</p>